### ISSUE TITLE
How to update User email signature

### ISSUE THREAD

**Issue by ursoforte** [infusionsoft/infusionsoft-php#285]
Hi Team,

I found an endpoint in the API to capture the user's email signature. But there is no method to update.
Here: https://developer.infusionsoft.com/docs/rest/#!/Users/getUserSignatureUsingGET

Also, the PHP SKD does not have a method to it.
Please, how do I update an email signature to the user via API? My final goal is to integrate Keap with Bybrand.

Thank you.

**Comment by skeemer** [comment]
I don't believe this is currently possible. There is no user updating in the REST APIs or XML-RPC API.
